practice-two/README.md

```diff
@@ -1 +1,42 @@
-# JavaScript
\ No newline at end of file
+# Practice two
+> A simple page.
+## Getting Started 
+
+### Design
+Sample design: [here](https://1.bp.blogspot.com/--IZPk65USOk/X7TPu3ZDy_I/AAAAAAAAIZI/YM2o886lyCMzLbVG8iVm0QilX-SFHq3NQCLcBGAsYHQ/s1532/javascript-crud-example-list-page.png)
+
+### Prerequisites
+Before you continue, ensure you meet the following requirements:
+* You have installed the latest VScode.
+* Use the latest browser version (Chrome, Firefox, MS Edge)
+
+### Requirement
+* Apply the knowledge of HTML/CSS/JavaScript to build a simple page.
+* Manipulate table and form with DOM.
+* Store data using localStorage.
+* Improve code styles.
+  
+### Installation
+* You can download my code [here](https://github.com/jinety/JavaScript/tree/feature/practice-two)
```

Comment:
This is not in Installation. Actually, it's duplicated Clone Repository  in Executing program